outport/process/transactionsfee/transactionsFeeProcessor.go

```diff
@@ -37,6 +40,7 @@ func NewTransactionsFeeProcessor(arg ArgTransactionsFeeProcessor) (*transactions
 		txFeeCalculator:  arg.TxFeeCalculator,
 		shardCoordinator: arg.ShardCoordinator,
 		txGetter:         newTxGetter(arg.TransactionsStorer, arg.Marshaller),
+		log:              logger.GetOrCreate("outport/process/transactionsfee"),
```

Comment:
we can either have the constant string defined as a constant or the log instance declared as a var to comply with the rest of the project.